src/FsAutoComplete.Core/UntypedAstUtils.fs

```diff
@@ -445,9 +445,9 @@ module Syntax =
       match s with
       | SynMemberDefn.AbstractSlot(valSig, _, _, _) -> walkValSig valSig
       | SynMemberDefn.Member(binding, _) -> walkBinding binding
-      | SynMemberDefn.ImplicitCtor(_, AllAttrs attrs, AllSimplePats pats, _, _, _, _) ->
+      | SynMemberDefn.ImplicitCtor(attributes = AllAttrs attrs; ctorArgs = ctorPattern) ->
```

Comment:
Implicit constructors now are able to have more patterns, so we no longer have a SynSimplePats, we have a single SynPat that we have to traverse.